microdata_tools/_decrypt.py

```diff
@@ -18,9 +19,18 @@
 
 def decrypt(rsa_keys_dir: Path, dataset_dir: Path, output_dir: Path):
     dataset_name = dataset_dir.stem
-    encrypted_csv_file = Path(dataset_dir / f"{dataset_name}.csv.encr")
+    encrypted_csv_file = Path(dataset_dir / f"{dataset_name}_chunk_1.csv.encr")
     output_dataset_dir = output_dir / dataset_name
 
+    if not output_dataset_dir.exists():
+        os.makedirs(output_dataset_dir)
+
+        # if chunk_dir.exists():
+        #     logger.info(f"Encrypted file found in {dataset_dir}")
+
+        # Create temp directory for decrypted chunks
+        decrypted_dir = output_dir / "decrypted"
```

Comment:
I assume the output_dir will be passed as the WORKING_DIR in the job_executor. Since there are possibly 4 simultaneous jobs running they will collide when all putting their files in WOKRING_DIR/decrypted

Putting the descrypted dir under output_dataset_dir instead of directly under output_dir would resolve this 👍🏻

Reply:
Yes it should use output_dataset_dir and not output_dir
```py
output_dataset_dir = output_dir / dataset_name
```